cmake/linux.cmake

```diff
@@ -54,7 +54,6 @@ endif()
 message(STATUS "${CURRENT_EXTERNAL_PROJECT}<${CMAKE_BUILD_TYPE}>: Done.")
 
 # configure openssl
-set(OPENSSL_USE_STATIC_LIBS TRUE)
```

Comment:
I'm not really sure why this is, but it did not find openssl on my machine unless I remove this line. Is there a reason we look for static libs here?

Reply:
This is probably to minimize runtime dependencies. In general, we are using static libraries to avoid version mismatches across platforms. But since openssl is only required on linux, it is probably safe to use the shared libraries.